mobile/lib/features/wallet/wallet_change_notifier.dart

```diff
@@ -29,6 +33,14 @@ class WalletChangeNotifier extends ChangeNotifier implements Subscriber {
     super.notifyListeners();
   }
 
+  Future<void> initialize() async {
+    await refreshWalletInfo();
+
+    timer = Timer.periodic(const Duration(seconds: 30), (Timer t) async {
+      await refreshWalletInfo();
```

Comment:
I'm not sure if it does, but if `refreshWalletInfo` triggers an on-chain sync, than this would be too often.